### PR TITLE
Add OpInsel

### PR DESCRIPTION
**Pull request by Nils-afk**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/opinsel/background.png
Comment (DoJapHD) [inline minecraft_servers/opinsel/background.png:1]: Compress

File: minecraft_servers/opinsel/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/opinsel/background@2x.png:1]: Compress

File: minecraft_servers/opinsel/manifest.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "server_wildcards": [
+      "%.opinsel.de"
+    ],
+    "supported_languages": [
+      "de"
+    ],
+    "social": {
+      "web": "https://opinsel.de",
+      "web_shop": "https://opinsel.de/shop",
+      "discord": "https://opinsel.de/discord"
```
Comment (DoJapHD) [inline minecraft_servers/opinsel/manifest.json:14]: Custom invites reserved for partners; use https://discord.gg/[INVITE]

File: minecraft_servers/opinsel/manifest.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    "location": {
+      "country": "Germany",
+      "country_code": "DE"
+    },
+    "gamemodes": {
+      "lobby": {
```
Comment (DoJapHD) [inline minecraft_servers/opinsel/manifest.json:21]: Not really a gamemode

File: minecraft_servers/opinsel/manifest.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        "name": "Lobby", 
+        "color": "#52ff59",
+        "command": "/warp Spawn"
+      },
+      "stats": {
```
Comment (DoJapHD) [inline minecraft_servers/opinsel/manifest.json:26]: Also not really a gamemode

File: minecraft_servers/opinsel/manifest.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+    "server_name": "opinsel",
+    "nice_name": "OpInsel",
+    "direct_ip": "mc.opinsel.de",
+    "server_wildcards": [
+      "%.opinsel.de"
```
Comment (DoJapHD) [inline minecraft_servers/opinsel/manifest.json:6]: Do you really have a wildcard?
E.g. no DNS record for "abc.opinsel.de" found

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/opinsel/manifest.json (hunk 1 above)
Comment (Nils-afk) [reply minecraft_servers/opinsel/manifest.json:14]: https://discord.com/invite/Mn9drhfaDQ

File: minecraft_servers/opinsel/manifest.json (hunk 2 above)
Comment (Nils-afk) [reply minecraft_servers/opinsel/manifest.json:21]: Can be removed

File: minecraft_servers/opinsel/manifest.json (hunk 3 above)
Comment (Nils-afk) [reply minecraft_servers/opinsel/manifest.json:26]: Can be removed

File: minecraft_servers/opinsel/manifest.json (hunk 4 above)
Comment (Nils-afk) [reply minecraft_servers/opinsel/manifest.json:6]: Can be removed

**Review by Nils-afk — COMMENTED** [review]
Changed

**Review by Nils-afk — COMMENTED** [review]
Fixed error

**Review by Nils-afk — COMMENTED** [review]
Fixed error

**Comment by Nils-afk** [comment]
Fixed everything

**Comment by Nils-afk** [comment]
@DoJapHD Done

**Comment by maaaathis** [comment]
You don't always have to close a PR and open a new one btw. As long as you commit your changes to the same branch in your repo, the PR will update automatically.

**Comment by Nils-afk** [comment]
> You don't always have to close a PR and open a new one btw. As long as you commit your changes to the same branch in your repo, the PR will update automatically.

Ok Thanks for the information.

**Comment by Nils-afk** [comment]
> Images still need to be compressed

I have already compressed these.

**Review by DoJapHD — APPROVED** [review]
There we go, now they are compressed.

**Pull request merged**